RecoJets/JetPlusTracks/plugins/JetPlusTrackProducerAA.cc

```diff
@@ -158,13 +157,13 @@ void JetPlusTrackProducerAA::produce(edm::Event& iEvent, const edm::EventSetup&
     bool ok = false;
```

Comment:
could you change the variable name to something more self-explanatory? Thanks

```suggestion
    bool validMatches = false;
```